DisguiseUnityRenderStream/PluginEntry.cs

```diff
@@ -502,6 +504,34 @@ public RS_ERROR getFrameParameters(UInt64 schemaHash, ref float[] outParameterDa
             }
             //return RS_ERROR.RS_ERROR_UNSPECIFIED;
         }
+        
+        public RS_ERROR GetFrameParameters(UInt64 schemaHash, ref NativeArray<float> outParameterData)
```

Comment:
Minor: Are the old getFrameParameters and getFrameImageData (lower case first letter) still used?  If no I guess we could remove it.